src/starter-kit wiki page folder/Publications.md

```diff
@@ -0,0 +1,19 @@
+## 2018

```

Comment:
Instead of adding this file I would replace the content of the [`src/publications.md`](https://github.com/bids-standard/bids-starter-kit/blob/main/src/publications.md) file with the content of this one.

Reply:
Sure, will do.